Kernel/Memory/MemoryManager.cpp

```diff
@@ -989,9 +996,7 @@ ErrorOr<NonnullRefPtrVector<PhysicalPage>> MemoryManager::allocate_contiguous_us
             return physical_pages;
         }
     }
-
-    dmesgln("MM: no contiguous user physical pages available");
-    return ENOMEM;
+    VERIFY_NOT_REACHED();
```

Comment:
Why is this now a verify? What if there's enough pages available, but they're not contiguous?

Reply:
That's a good point, I didn't think of that, will update